### ISSUE TITLE
Windows: Cody webview load failure caused by invalid usage of `vscode.Uri.parse` with `extensionPath`

### ISSUE THREAD

**Issue by Fire-** [sourcegraph/sourcegraph#50142]
- **Sourcegraph version:**  Cody
- **Platform information:**  Windows 10; VSCode 1.76.2

#### Steps to reproduce:

1. Install the Sourcegraph Cody extension
2. Attempt to open chat

#### Actual behavior: The view fails to load, displaying only "An error occurred while loading view: cody.chat"

![image](https://user-images.githubusercontent.com/1456217/228692634-4c48f945-3172-4ed4-8cb0-653d8115397b.png)

#### Additional Information: The extension is failing to load its index.html from disk:

```log
  ERR Unable to resolve filesystem provider with relative file path 'c:\Users\fire\.vscode\extensions\sourcegraph.cody-ai-0.0.3/dist/index.html': Error (FileSystemError): Unable to resolve filesystem provider with relative file path 'c:\Users\fire\.vscode\extensions\sourcegraph.cody-ai-0.0.3/dist/index.html'
	at Yt.c (c:\Users\fire\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:98:26384)
	at Object.readFile (c:\Users\fire\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:98:24937)
	at async rn.resolveWebviewView (c:\Users\fire\.vscode\extensions\sourcegraph.cody-ai-0.0.3\dist\extension.js:83529:11)
	at async C.$resolveWebviewView (c:\Users\fire\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:98:179830)
```

This is due to the extension currently using [`vscode.Uri.parse`](https://code.visualstudio.com/api/references/vscode-api#Uri.parse) for a file path rather than [`vscode.Uri.file`](https://code.visualstudio.com/api/references/vscode-api#Uri.file); Basically, we got lucky that `parse` wasn't breaking this more widely already. That said, the better thing to do here is swapping out the `context.extensionPath` in `CommandsProvider` with `context.extensionUri`, which fixes the issue and improves compatibility with Web Extensions

**Comment by christian-korneck** [comment]
It appears the Cody vscode extension is currently broken on Windows due to this. Tried on several hosts.

**Comment by Fire-** [comment]
I'm working on fixing this btw

**Comment by jdorfman** [comment]
Thanks @Fire- let us know if you run into any issues

**Comment by christian-korneck** [comment]
I believe someone already fixed this in #50210 and in the meantime it found it’s way into the marketplace release. It seems to be working for me now.

**Comment by Fire-** [comment]
good to know; I haven't had the time free to tie up my branch, so I'll probably rebase and change focus to the context changes so Cody can function as a Web Extension

**Comment by glokta1** [comment]
Not sure if this is related but on Linux, I don't get the error message but the chat is just blank: 
![image](https://github.com/sourcegraph/sourcegraph/assets/71697384/0ab362c7-a745-408a-a5fd-2643fa69dc1e)

**System**
VSCodium Version 1.78.1
Linux x64 5.15.109.1-MANJARO

**Comment by glokta1** [comment]
Okay it's fixed now. It was a problem with webiew in VSCodium 1.78.1 which was fixed in 1.78.2 https://github.com/VSCodium/vscodium/pull/1514

**Comment by jdorfman** [comment]
@glokta1 ok I will close this issue. Open a new one if you run into further issues. :)
